src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -169,7 +169,7 @@ private void Analyze(TOptions options, AggregatingLogger logger)
             targets = ValidateTargetsExist(_rootContext, targets);
 
             // 5. Initialize report file, if configured.
-            InitializeOutputFile(options, _rootContext, targets);
```

Comment:
>targets [](http://example.com/codeflow?start=56&length=7)

Revert this change please it's unnecessarily breaking customers.

Reply:
This API isn't exposed to customers. Only tools that use it will see a difference.